grammars/css.cson

```diff
@@ -561,6 +561,44 @@
         'captures':
           '1':
             'name': 'punctuation.definition.entity.css'
+          '2':
+            'name': 'entity.other.namespace-prefix.css'
+          '3':
+            'name': 'punctuation.separator.css'
+          '4':
+            'name': 'entity.other.attribute-name.css'
+          '5':
+            'name': 'keyword.operator.pattern.css'
+          '6':
+            'name': 'string.unquoted.attribute-value.css'
+          '7':
+            'name': 'string.quoted.attribute-value.css'
+          '8':
+            'name': 'punctuation.definition.string.begin.css'
+          '9':
+            'name': 'punctuation.definition.string.end.css'
+          '10':
+            'name': 'punctuation.definition.entity.css'
+        'match': '''(?x)
+          (\\[)\\s*
+          (?:(\\g<ident>|\\*)?(\\|))?     # qualified name prefix
+          ([\\w:&&[^0-9]][\\w:.-]*+)\\s*  # attribute name
+          (?:
+            ([~|^$*]?=)\\s*
+            (?:                           # attribute value
+              (\\g<ident>)
```

Comment:
I don't think these lines are correct.  I don't see anywhere in [the spec](https://drafts.csswg.org/selectors-3/#attribute-selectors) (which you also seem to have used) that if the value is exactly the ident then it should be treated specially.

Or maybe I'm misinterpreting this line and the meaning of `\g`?